### ISSUE TITLE
What should I do with PR after resignation?

### ISSUE THREAD

**Issue by vassilevsky** [zerocracy/zerocracy.github.io#99]
* I received a job (https://github.com/g4s8/gitstrap/issues/9)
* I created a PR for it (https://github.com/g4s8/gitstrap/pull/19)
* The PR needs work
* The job was taken away from me in 10 days and I got -30 points

What do I do with the PR now?

If the PR is merged, will I get the 30 points back? (I know that I will get no money.)

There is nothing about this in the Policy.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@vassilevsky[/z](https://www.0crat.com/u/vassilevsky) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@vassilevsky the only thing you can do is to ask ARC to assign you back to the task. When your task needs additional time, e.g. when you're making code-review changes you can register impediment to postpone the task and temporary disable 10 days rule, see https://www.zerocracy.com/policy.html#9

**Comment by vassilevsky** [comment]
Yeah I didn't know about the `wait` command in the beginning, but then I read about it and will use.

So, after resignation, even if the PR is merged, I get no points since I'm no longer assigned to the job?

**Comment by g4s8** [comment]
@vassilevsky you'll get points and payment if you are ticket assignee.
